using-timescaledb/custom-jobs.md

```diff
@@ -0,0 +1,221 @@
+# Custom Jobs
+
+Custom jobs allow you to run functions and procedures implemented in a
+language of your choosing on a schedule within TimescaleDB. This allows
+automatic periodic tasks that are not covered by existing policies and
+even enhancing existing policies with additional functionality.
+
+### Creating a custom job procedure [](create-procedure)
+
+The signature for custom jobs is (job_id INT, config JSONB).
+The content of the config JSONB is completely up to the job and may
+also be NULL if no variables are required.
+
+Template for a custom job procedure.
+
+```sql
+CREATE OR REPLACE PROCEDURE custom_job(job_id int, config jsonb) LANGUAGE PLPGSQL AS
+$$
+BEGIN
+	RAISE NOTICE 'Executing job % with config %', job_id, config;
+END
+$$;
+```
+
+### Registering your custom job [](register)
+
+When registering the custom job `job_id` and `config` are stored in the
+TimescaleDB catalog. The `config` JSONB can be modified with `alter_job`.
+`job_id` and `config` will be passed as arguments when the procedure is
+executed as background process or when run with `run_job`.
+
+```sql
+SELECT add_job('custom_job','1h');
+```
+
+Register the created job with the automation framework. `add_job` returns the job_id
+which can be used to execute the job manually with `run_job`.
+
+```sql
+CALL run_job(1000);
+```
+
+Execute job with id 1000 in current session.
+
```

Comment:
Could you also add an example on how to view these jobs by querying timescaledb_information.xxxx view. Either as a separate section or as another SQL stmt here.

Reply:
Please briefly explain use of `CALL` vs. `SELECT`, given that most functions use select instead.

Reply:
Separately, it seems odd that `run_job` is introduced under `add_job`, then there is a fairly empty section called "Testing/Debugging".  Making an alternative suggestion.